code/datums/ai/dog/dog_controller.dm

```diff
@@ -97,6 +97,10 @@
 /datum/ai_controller/dog/proc/on_attack_hand(datum/source, mob/living/user)
 	SIGNAL_HANDLER
 
+	var/mob/living/living_pawn = pawn
+	if(living_pawn.stat == DEAD)
+		to_chat(user, span_warning("When you touch [pawn], you feel a dead cold from their body."))
```

Comment:
This reads kinda awkwardly, can you rephrase?

Reply:
ALSO, we have that not-so-secret feature in the codebase where clowns don't understand death. Might be nice to have a check for that :)

Reply:
it may be worth deleting this message in the chat, I doubt the need for this.